src/popup/extension-status-manager.ts

```diff
@@ -2,33 +2,32 @@
 import { EXTENSION_SETTINGS_CHANGE, ExtensionSettings } from 'shared/extension';
 import { POPUP_TO_WEBPAGE_MESSAGE } from 'shared/messaging';
 
+const STORAGE_KEY = 'extension-status';
 export class ExtensionStatusManager {
-  private static STORAGE_KEY = 'extension-status';
-
   static enable() {
     chrome.storage.local
       .set({
-        [this.STORAGE_KEY]: true,
+        [STORAGE_KEY]: true,
```

Comment:
@SolutionsEngineer I removed `this` in this class because at some point in runtime I saw that it was showing `undefined`, maybe due to the fact that it is static class.